src/views/Styleguide.vue

```diff
@@ -1,7 +1,11 @@
 <template>
-  <div>
-    <div>{{ $t('styleguide') }}!</div>
+<div class="p-20">
+  <div class="buttonstyle">
+  <h1 class="text-5xl font-bold my-4">BaseButtonStyles</h1>
+  <base-button-styles></base-button-styles>
+  </div>
 
+    
```

Comment:
Add a title for this section 
`<h1 class="text-5xl font-bold my-4">BaseCard Styles</h1>`